lib/stack_master/template_compiler.rb

```diff
@@ -4,7 +4,7 @@ class TemplateCompiler
 
     def self.compile(config, template_file_path)
       template_compiler_for_file(template_file_path, config).compile(template_file_path)
-    rescue Exception
+    rescue StandardError
```

Comment:
`StandardError` can be omitted because it's the default when using `rescue` without an error class